docs/guides/signals.md

```diff
@@ -46,7 +46,9 @@ The method returns `True` (indicating that all intervals are ready for evaluatio
 ```python linenums="1"
 import random
 import typing as t
-from sqlmesh.core.scheduler import signal_factory, Batch, Signal
+from sqlmesh.core.scheduler import signal_factory, Signal
+from sqlmesh.utils.date import DatetimeRanges
```

Comment:
So this is a breaking change technically?

Reply:
Yes the import path does change but the functionality doesn't. Changed title to be breaking.